### DISCUSSION TITLE
Languages

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [koreader/koreader#8151] [Q&A]
When I change the language to German, it says I should restart the app. But when I do that, it's English again. What could be the reason for that? Or is there something else that is not translated?

**Comment by NiLuJe** [comment]
On which platform?

**Reply by mondlicht-und-sterne** [reply]
https://f-droid.org/packages/org.koreader.launcher.fdroid/
this one of course

**Reply by NiLuJe** [reply]
Does it work with our own builds?

I mean, i can see the l10n stuff in the source tarball, but something might have gone screwy somewhere along the way...

**Reply by NiLuJe** [reply]
And, not that it should matter much, but, again, who knows: on which device?

**Reply by mondlicht-und-sterne** [reply]
My device is an Samsung Galaxy S21 5G, Android 11

I'll have to try that with your builds - normally I only install from f-droid.

**Reply by Frenzie** [reply]
It works for me with the F-Droid 2021.08 build on my Realme 7 Pro, Android 11.

**Comment by mondlicht-und-sterne** [comment]
It's the same problem, so it's not that. can i also take the f-droid version.
but it only works in english.

**Reply by NiLuJe** [reply]
FWIW, it works just fine over here on a OnePlus 6 on Android 11 (the "Please restart" popup should in fact be translated even before the restart).

Not quite sure what could help debug this further, as the i18n stuff is pretty low-level :?

Let's wait to see if our Android wizards have any better ideas ;).

**Reply by mondlicht-und-sterne** [reply]
Okay then, I'll just wait.

This does not fit in here with the "Languages" topic I started.
In my spare time I paint F-Droid app logos and promote them on Mastodon and pixelfed.
And I make exhibitions. I already have one in Berlin / Germany, the second one is currently being planned.
I haven't painted koreader yet. the new app logo is now blue and no longer turquoise, right?

**Reply by NiLuJe** [reply]
The color simply denotes the build flavor: ours is still turquoise, F-Droid is blue, and debug builds are orange ;).

**Reply by mondlicht-und-sterne** [reply]
Good to know. So turquoise, thank you

**Comment by pazos** [comment]
> When I change the language to German, it says I should restart the app. But when I do that, it's English again. What could be the reason for that? Or is there something else that is not translated?

In case you're killing the app from the recents menu of android try with a clean exit: `hamburger menu - > exit`. If you can reproduce with a clean exit I have no idea.
